src/include/cursor.i

```diff
@@ -139,6 +139,70 @@ __curfile_leave(WT_CURSOR_BTREE *cbt)
 }
 
 /*
+ * __wt_curindex_get_value_ap --
```

Comment:
Can this please be renamed `__wt_curindex_get_valuev` (matching `__wt_cursor_get_valuev` for the `va_list` version)?